hr_timesheet_rounded/models/project_project.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo import fields, models
+
+
+class ProjectProject(models.Model):
+
+    _inherit = 'project.project'
+
+    timesheet_rounding_granularity = fields.Float(
+        string='Timesheet rounding granularity',
+        default=0.25,  # 15 min
+        help="""1.0 = hour
+            0.25 = 15 min
+            0.084 ~= 5 min
+            0.017 ~= 1 min
+            """
+    )
+    timesheet_rounding_method = fields.Selection(
+        string='Timesheet rounding method',
+        selection=[
+            ('HALF_UP', 'Closest'),
+            ('UP', 'Up'),
```

Comment:
Any reason not to allow `DOWN` rounding ?

Reply:
usually you don't want to round DOWN: you'll get less money :wink:

Reply:
That depends how you intend to use the module...
E.g. I could conceive some companies must round down because of specific contracts/projects and IMO it's not a good enough reason to restrict such a possibility in an OCA module. :roll_eyes:

Reply:
You may also be interested in https://github.com/OCA/timesheet/pull/139

Reply:
@alexey-pelykh I had a quick look at your PR. Seems the two modules can coexist since the purpose is different: we want only to affect invoicing, not the real unit amount tracked internally.
All in all I think @grindtildeath [here](https://github.com/OCA/timesheet/pull/257#issuecomment-493143376) is right and we should rename to something more sale-related.

Reply:
Thanks for the feedback!